package.json

```diff
@@ -9,7 +9,7 @@
     "StrongLoop"
   ],
   "engines": {
-    "node": ">=4.0.0"
+    "node": ">=6"
```

Comment:
Please don't mix changes dropping support for older versions of Node with an update of tests. There should be two commits:
 1. Drop support for Node.js 4.x
 2. Update help messages expected by tests

Reply:
done